easybuild/easyconfigs/o/OpenMPI/OpenMPI-2.0.1-iccifort-2017.1.132-GCC-5.4.0-2.26.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'ConfigureMake'
+
+name = 'OpenMPI'
+version = '2.0.1'
+
+homepage = 'http://www.open-mpi.org/'
+description = """The Open MPI Project is an open source MPI-2 implementation."""
+
+toolchain = {'name': 'iccifort', 'version': '2017.1.132-GCC-5.4.0-2.26'}
+
+sources = [SOURCELOWER_TAR_GZ]
+
+source_urls = ['http://www.open-mpi.org/software/ompi/v%(version_major_minor)s/downloads',]
+
+#patches = [
+#    'OpenMPI-%(version)s_common-cuda-lib.patch',
+#    'OpenMPI-%(version)s-vt_cupti_events.patch',
+#]
+
+dependencies = [('hwloc', '1.11.4')]
+
+configopts = '--with-threads=posix --enable-shared --with-verbs '
```

Comment:
Yeah, enable-mpi-thread-multiple doesn't work on IB (still, i wish it could get fixed) so it's a local mod that i forgto to revert. I'll try to get some work done on the above today...